benchmarks/synchrobench/src/test/java/com/yahoo/oak/OakMyBufferTest.java

```diff
@@ -48,6 +48,21 @@ public void testPut() {
         }
     }
 
+
+    @Test
+    public void testIncreasePut() {
+        Integer v =  0;
+        for (long i = 1000000; i > 0; ) {
+            v++;
+            MyBuffer key = new MyBuffer(Parameters.keySize);
+            key.buffer.putInt(0, v);
+            MyBuffer val = new MyBuffer(Parameters.valSize);
+            val.buffer.putInt(0, v);
+            oakBench.putOak(key, val);
+            i--;
+        }
+    }
```

Comment:
Why do you iterate backward over `i` when you don't even use it?
It would be simpler to iterate forward over `v`.

Reply:
fixed